src/pool/srv_pool.c

```diff
@@ -60,6 +60,7 @@ struct pool_svc {
 	rdb_path_t		ps_handles;	/* pool handle KVS */
 	rdb_path_t		ps_user;	/* pool user attributes KVS */
 	struct ds_pool	       *ps_pool;
+	uint32_t		ps_disable_evict:1; /* ignore evict event */
```

Comment:
I suppose it should be configured through pool prop DAOS_PROP_PO_SELF_HEAL, it's better to use a name like 'manual_exclude' or 'auto_exclude'.

Reply:
oh, this enable/disable thing I will deal with it in next patch.

Reply:
so shouldn't this auto exclude be disabled by default for this moment? otherwise, a small network hiccup could cause targets excluding?

Reply:
I thought swim should avoiding this, i.e. it will call these event callback only if it is certain these nodes are "DEAD"

Reply:
This enable/disable thing is actually suppose to be manual in our case. i.e. I will provide an API to enable/disable event callback. So the control panel can use it during shutdown.